options/rtdl/generic/main.cpp

```diff
@@ -46,7 +46,7 @@ DebugInterface globalDebugInterface;
 
 // Use a PC-relative instruction sequence to find our runtime load address.
 uintptr_t getLdsoBase() {
-#if defined(__x86_64__) || defined(__x86_64__) || defined(__aarch64__)
+#if defined(__x86_64__) || defined(__i686__) || defined(__aarch64__)
```

Comment:
Is this actually needed to get static builds working?

Also, this commit should be squashed with the other one

Reply:
Actually no, it is not needed. I'll probably drop them.